properties.py

```diff
@@ -1016,7 +1021,7 @@ def update_area_shape(self, context):
 
     shadingrate = FloatProperty(
         name="Light Shading Rate",
-        description="Shading Rate for lights.  Keep this high unless needed for using detailed maps",
+        description="Shading Rate for lights.  Keep this high unless banding is noticed or pixellation on detailed maps",
```

Comment:
maybe says this applies to lights with maps?   Maybe a grammar change:  Keep this high unless banding or pixellation on detailed  light maps